src/selectedtests/app/app.py

```diff
@@ -37,6 +38,13 @@ def create_app(mongo_wrapper: MongoWrapper, evg_api: EvergreenApi) -> FastAPI:
         prefix="/projects/{project}/test-mappings",
         tags=["projects"],
     )
-    app.state.db = mongo_wrapper
-    app.state.evg_api = evg_api
+
+    def dependencies(binder: inject.Binder):
```

Comment:
So I'm wondering about that -- can we not just pass around the dependencies like normal in the *_cli.py scripts? One of the goals for the exercise was to only need to do DI in the app, and not have to fixup the cli scripts (migrating in pieces has planning/risk advantages).

I tried to get their tests working without using DI on them -- are there other cases I should ensure still work?

Reply:
RE migrating in pieces, I see. Yeah I think if we were going to switch to the `inject` library we should do it in pieces, that makes sense.

Since the web app tests pass, and you only changed the web app code, I don't think there are other pieces that need to be tested.